roles/oneview_server_profile/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for oneview_server_profile
+config: "~/.ansible/collections/ansible_collections/hpe/oneview/roles/oneview_fc_network/files/oneview_config.json"
```

Comment:
replace oneview_fc_network with oneview_server_profile.
Ideally it should be this resource name
```
config: "~/.ansible/collections/ansible_collections/hpe/oneview/roles/oneview_fc_network/files/oneview_config.json"
```

Reply:
done